src/test/scala/uk/gov/hmrc/test/ui/pages/CommonElements.scala

```diff
@@ -35,7 +35,8 @@ trait CommonElements extends BasePage with Matchers{
     //Dividends Elements
     "dividends"                 -> By.id("dividends_link"),
     "UK dividends amount"       -> By.id("amount"),
-    "UK other dividends amount" -> By.id("amount")
+    "UK other dividends amount" -> By.id("amount"),
+    "UK dividends?" -> By.cssSelector("#main-content > div > div > dl > div > dt")
```

Comment:
We don't need to verify text here, the fact the it takes you to the CYA page should be sufficient for a journey test.